integrations/symfony-bundle.rst

```diff
@@ -91,22 +91,25 @@ You can configure your clients with default options. These default values will b
                         headers:
                             Content-Type: 'application/json'
             acme:
-                factory: 'httplug.factory.guzzle6'
+                factory: 'httplug.factory.curl'
                 config:
-                    base_uri: 'http://google.se/'
+                    78: 4 #CURLOPT_CONNECTTIMEOUT
```

Comment:
could we use expression language for this? hm, i fear not, looks like we can only use that for the value, but not for the key. only way would be to make it a php configuration file which is bad for everything else.